docroot/themes/custom/uids_base/templates/uids/card.html.twig

```diff
@@ -1,32 +1,68 @@
-{{ attach_library('uids_base/card-block') }}
+{{ attach_library('uids_base/card') }}
 {% set attributes = attributes ?: create_attribute() %}
 {% set card_classes = ['card'] %}
 
+{# The following code is to determine where a link should be attached, if it exists. #}
+{% set link_element, show_button = false, false %}
+{% if card_link_url is not empty %}
+  {% if card_title is not empty %}
+    {# Title is set, use it as the linking element.  #}
+    {% set link_element = 'title' %}
+    {# If we also have link text, then we'll show a psuedo-button. #}
+    {% if card_link_title is not empty %}
+      {% set show_button = true %}
+    {% endif %}
+  {% elseif card_link_title is not empty %}
+    {# Title is not set, but there is link text, so use that instead. #}
+    {% set link_element = 'button' %}
+    {# Show an actual link button. #}
+    {% set show_button = true %}
+  {% elseif card_image is not empty %}
+    {# If the image exists, use that as the linking element. #}
+    {% set link_element = 'image' %}
+  {% else %}
+    {# Nothing else to use, so wrap the whole card. #}
+    {% set link_element = 'card' %}
+  {% endif %}
+{% endif %}
+
+{# Do not show the aria-describedby attribute if title AND link text are not both present. #}
+{% if link_element != 'title' or not show_button %}
+{% set card_aria_describedby = '' %}
+{% endif %}
+
 <div{{ attributes.addclass(card_classes) }}>
+  {{ title_prefix }}
+  {{ title_suffix }}
   {% block card_img %}
     {% if card_image is not empty %}
       {# @todo Add image template #}
       <div class="card__media">
-        <img class="card__img" src="{{ card_image }}" alt="Alt Text">
+        {% if link_element == 'image' %}
+        <a href="{{ card_link_url }}">
+          {% endif %}
+          <img class="card__img" src="{{ card_image }}" alt="Alt Text">
```

Comment:
alt text is hardcoded here

Reply:
I changed this so that it just prints the `card_image` value in this template, since that is what happens in all our instance template and it allowed me to remove the `card_img` block from those templates.